viadot/flows/sql_server_to_parquet.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List, Literal
+
+from prefect import Flow
+
+from viadot.task_utils import df_to_parquet
+from viadot.tasks import SQLServerToDF
+
+
+class SQLServerToParquet(Flow):
+    def __init__(
+        self,
+        name,
+        sql_query: str,
+        local_file_path: str,
+        sqlserver_config_key: str = None,
+        if_exists: Literal["fail", "replace", "append", "skip", "delete"] = "fail",
+        timeout: int = 3600,
+        *args: List[any],
+        **kwargs: Dict[str, Any],
+    ):
+        """
+        Flow for upolading data from SQL Server to Parquet file.
+
+        Args:
+            name (str): The name of the flow.
+            sql_query (str, required): The query to execute on the SQL Server database. If don't start with "SELECT"
```

Comment:
Is writing `required` mandatory? 
For `name` and `local_file_path` you have only type.

Reply:
I added `required` for `name` and `local_file_path`